.goreleaser.yml

```diff
@@ -6,9 +6,15 @@ release:
         name: driftctl
     draft: false
     prerelease: auto
+    extra_files:
+        - glob: ./acknowledgements.tar.gz
 before:
     hooks:
         - go generate ./...
+        - go get github.com/google/go-licenses
```

Comment:
I think it's more correct to use `go install github.com/google/go-licenses@latest` here.

https://go.dev/ref/mod#go-install

Reply:
funny, didn't know go get will get completely removed https://go.dev/doc/go-get-install-deprecation